### ISSUE TITLE
Timestamp with time zone issue in field of jsonb body

### ISSUE THREAD

**Issue by eduardomourar** [dmfay/massive-js#563]
Hello,

I am having an issue with massivejs v2.7.3 while running the following query:
```
db.example.findDoc({
		"endTimestamp >=": fromDateTime.toDate(),
	},
	{
		order: [
		  {field: "body->>'endTimestamp'", direction: "desc", type: "timestamptz"},
		]
	}, function (err, result) {}
)
```

I found out that in the code [here](https://github.com/dmfay/massive-js/blob/f9684ac37e4af870ae095c3b234102a5520c1e2a/lib/where.js#L199), the date field is being cast to `timestamp`. I have the timestamp inside the jsonb body with timezone. When I run this, the time zone is ignored and uses the one from the system, of course.

We should be using `timestamptz` here instead, right? After doing this change, it works as expected.

Let me know if this makes sense and I will create a PR with this small change.

**Comment by dmfay** [comment]
Yeah, a `timestamp` is appropriate for certain situations but I'm comfortable calling that a bug since almost everyone will expect a `timestamptz`. Go ahead and make the change (add a test too please!) and I'll cut 2.7.4.

**Comment by dmfay** [comment]
I've published 2.7.4 on the `legacy` tag and 4.7.1 on `latest`. Thanks!
